index.html

```diff
@@ -158,6 +159,22 @@
 if (localStorage.getItem("writing") !== null) {

```

Comment:
Should we refactor this javascript code into it's own file?

Reply:
I know it's bad practice, but I'm much into small minimalist tools, and here I'd like to keep a single index.html containing css + html + js (for now).

Reply:
Lol. Your project will grow in due time because it's useful. Don't fall into this idea of "minimalism". It's impossible to maintain and makes it a pain for other developers.

Reply:
I agree to @lodenrogue since constant abstraction does not violate minimalism since it is essential